CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+# 0.19.3
+* More results highlight refactoring
```

Comment:
We should be more verbose here since we added stuff like auto-including highlight keys in context.include, etc